velox/connectors/hive/storage_adapters/gcs/examples/CMakeLists.txt

```diff
@@ -20,5 +20,6 @@ target_link_libraries(
   velox_gcs
   velox_core
   velox_hive_connector
+  velox_hive_iceberg_datasource
```

Comment:
These CMakeLists.txt additions should be redundant since `velox_hive_iceberg_datasource` is added to `velox_hive_connector`.

Reply:
@majetideepak It didn't work and would throw link errors. Do you know how to make velox_hive_connector contain the velox_hive_iceberg_datasource? I tried to add PUBLIC link lib to velox_hive_iceberg_datasource, but it also doesn't work.